[JBTM-3850] Disable not working XTS/ssl quickstart

marcosgopen opened the pull request:
https://issues.redhat.com/browse/JBTM-3850

Parent issue is https://issues.redhat.com/browse/JBTM-3849 

related to https://issues.redhat.com/browse/WFLY-18515

This quickstart will be disabled until https://issues.redhat.com/browse/JBTM-3851

Since the wsat-simple has been removed from wildfly/quickstart the ssl quickstart doesn't work anymore, so disabling it.

tomjenkinson commented:
Thank you for finding this @marcosgopen

(Please can you say if WFLY-18515 caused our build to fail? If not this is an opportunity to review that part and make sure it would fail.)

marcosgopen commented:
> Thank you for finding this @marcosgopen
> 
> (Please can you say if WFLY-18515 caused our build to fail? If not this is an opportunity to review that part and make sure it would fail.)

Yes, it does. E.g. this is the failing build: https://ci-jenkins-csb-narayana.apps.ocp-c1.prod.psi.redhat.com/view/Pull/job/btny-pulls-narayana-quickstart/323/console

marcosgopen commented:
@mmusgrov Do you think I need to create a Jira to create a new ssl quickstart?

mmusgrov reviewed (changes requested):
I'm all for removing WS quickstarts but we need a JBTM for it where we can justify it and track it. This qs resides in our repo and we are independent of WildFly. If the base WS functionality is still in narayana but WFLY no longer supports what we need then we either need justify why we are removing it from our repo or we need to investigate other platforms where we could run the quickstart.

marcosgopen commented:
> I'm all for removing WS quickstarts but we need a JBTM for it where we can justify it and track it. This qs resides in our repo and we are independent of WildFly. If the base WS functionality is still in narayana but WFLY no longer supports what we need then we either need justify why we are removing it from our repo or we need to investigate other platforms where we could run the quickstart.
> 
> @marcosgopen Can we find out why it was removed as out of date and whether or not there was a discussion about bringing it up to date?

Thanks Mike, I created the JBTM and asked to Eduardo. I will let you know and update the JBTM soon

mmusgrov commented:
But if we can't remove it immediately then it does need disabling in the pom until we can make a decision.

marcosgopen commented:
> But if we can't remove it immediately then it does need disabling in the pom until we can make a decision.

Thanks Mike, I changed the description of this PR, I updated the PR so that it disable the quickstart and I created an extra JBTM to update/remove the quickstart (see this PR description)

tomjenkinson reviewed (changes requested):
Actually I think if we are just disabling it, it's worth noting the reason in the https://github.com/jbosstm/quickstart/blob/main/XTS/ssl/README.md so that people don't try to run it manually.

marcosgopen commented:
> Actually I think if we are just disabling it, it's worth noting the reason in the https://github.com/jbosstm/quickstart/blob/main/XTS/ssl/README.md so that people don't try to run it manually.

Good point! Thanks @tomjenkinson !

tomjenkinson reviewed (approved):
Thank you!

mmusgrov reviewed (approved):
Looks good now, thanks.
